docs/source/tracking/autolog.rst

```diff
@@ -201,6 +201,25 @@ Also, MLflow will then automatically end the run once training ends via calls to
 
 If a run already exists when ``autolog()`` captures data, MLflow will log to that run but not automatically end that run after training. You will have to manually stop the run if you wish to start a new run context for logging to a new run.
 
+.. _autolog-langchain:
+
+LangChain
+---------
+
+Call the generic autolog function :py:func:`mlflow.langchain.autolog` before your training code to enable automatic logging of traces.
+See an example usage with `LangChain <https://github.com/mlflow/mlflow/tree/master/examples/langchain>`_.
+
+Autologging captures the following information:
+
++-----------+-------------+----------------+---------------+---------------------------------------------------------------------------------+
+| Framework | Metrics     | Parameters     | Tags          | Artifacts                                                                       |
++-----------+-------------+----------------+---------------+---------------------------------------------------------------------------------+
+| LangChain | --          | --             |  --           | - Traces                                                                        |
+|           |             |                |               | - `MLflow Model`_ (LangChain model) with model signature on training end        |
+|           |             |                |               | - Input example                                                                 |
+|           |             |                |               | - Datasets                                                                      |
```

Comment:
Honestly langchain autologging doesn't log datasets 😆 We should remove it from code as well

Reply:
oh! FYI @B-Step62 for the langchain doc page as well

Reply:
Ooh lol I didn't even realize I mentioned datasets there, it's almost auto-complete for me typing `models, datasets, etc...`. Will open a quick PR.